apps/docs/src/app/platform/component-docs/platform-search-field/platform-search-field-examples/platform-search-field-basic-example.component.scss

```diff
@@ -1,3 +1,4 @@
+@import "~@angular/cdk/overlay-prebuilt.css";
```

Comment:
please remove it
already fixed in this [PR](https://github.com/SAP/fundamental-ngx/pull/3985)